tox.ini

```diff
@@ -10,7 +10,16 @@ setenv =
     LC_CTYPE = en_US.UTF-8
 deps = -r{toxinidir}/dev-requirements.txt
 commands = py.test --timeout 300 []
-install_command = python -m pip install {opts} {packages}
+# Make sure the version of pip we're about to test is
+# not itself being used for setting up the environment.
```

Comment:
Okay - I see where you're coming from at this - I agree with you that using the code you want to test, to setup it's own test environment, is a little weird.

Changing the current setup to not do so is definitely an improvement. :)